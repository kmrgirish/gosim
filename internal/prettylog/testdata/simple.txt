=== RUN   TestLogForPrettyTest
{"time":"2020-01-15T14:10:03.000001234Z","level":"INFO","source":{"function":"translated/github.com/kmrgirish/gosim/internal_/tests/behavior_test.ImplTestLogForPrettyTest","file":"/Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/tests/behavior/log_gosim_test.go","line":53},"msg":"hello info","machine":"main","foo":"bar","goroutine":4,"step":1}
{"time":"2020-01-15T14:10:23.000001234Z","level":"INFO","source":{"function":"translated/github.com/kmrgirish/gosim/internal_/tests/behavior_test.ImplTestLogForPrettyTest.func1","file":"/Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/tests/behavior/log_gosim_test.go","line":58},"msg":"warn","machine":"machine-2","ok":"now","delay":0,"goroutine":5,"step":2}
{"time":"2020-01-15T14:10:23.000001234Z","level":"INFO","source":{"function":"translated/github.com/kmrgirish/gosim/internal_/tests/behavior_test.ImplTestLogForPrettyTest.func1","file":"/Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/tests/behavior/log_gosim_test.go","line":59},"msg":"before","machine":"machine-2","goroutine":5,"step":3}
{"time":"2020-01-15T06:10:23.000001234-08:00","level":"ERROR","source":{"function":"github.com/kmrgirish/gosim/gosimruntime.(*goroutine).step","file":"/Users/jelle/hack/gosim/gosimruntime/runtime.go","line":656},"msg":"uncaught panic","traceback":["goroutine 9 [running]:","github.com/kmrgirish/gosim/gosimruntime.(*goroutine).exitpoint(0x1400045de40)","  /Users/jelle/hack/gosim/gosimruntime/runtime.go:728 +0x9c","panic({0x1011dc360?, 0x10147b090?})","  /opt/homebrew/Cellar/go/1.23.3/libexec/src/runtime/panic.go:785 +0x124","translated/github.com/kmrgirish/gosim/internal_/tests/behavior_test.ImplTestLogForPrettyTest.func1()","  /Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/tests/behavior/log_gosim_test.go:60 +0xe0","translated/github.com/kmrgirish/gosim/internal_/simulation.(*Simulation).startMachine.func1()","  /Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/simulation/simulation_gosim.go:96 +0x4c","github.com/kmrgirish/gosim/gosimruntime.(*goroutine).entrypoint(0x1400045de40, 0x14000461f50)","  /Users/jelle/hack/gosim/gosimruntime/runtime.go:712 +0x60","github.com/kmrgirish/gosim/gosimruntime.goroutineEntrypoint()","  /Users/jelle/hack/gosim/gosimruntime/runtime.go:705 +0x30","github.com/kmrgirish/gosim/internal/coro.(*Coro).Start.func1(0x0?)","  /Users/jelle/hack/gosim/internal/coro/coro_linkname.go:38 +0x24","created by github.com/kmrgirish/gosim/internal/coro.(*Coro).Start in goroutine 5","  /Users/jelle/hack/gosim/internal/coro/coro_linkname.go:36 +0x70",""],"panic":"help","machine":"machine-2","goroutine":5}
