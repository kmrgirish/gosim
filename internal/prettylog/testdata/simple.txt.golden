=== RUN   TestLogForPrettyTest
    1 main/4     14:10:03.000 INF behavior/log_gosim_test.go:53 > hello info foo=bar
    2 machine-2/5 14:10:23.000 INF behavior/log_gosim_test.go:58 > warn delay=0 ok=now
    3 machine-2/5 14:10:23.000 INF behavior/log_gosim_test.go:59 > before
<nil> machine-2/5 14:10:23.000 ERR gosimruntime/runtime.go:656 > uncaught panic panic=help traceback=[
          "goroutine 9 [running]:",
          "github.com/kmrgirish/gosim/gosimruntime.(*goroutine).exitpoint(0x1400045de40)",
          "  /Users/jelle/hack/gosim/gosimruntime/runtime.go:728 +0x9c",
          "panic({0x1011dc360?, 0x10147b090?})",
          "  /opt/homebrew/Cellar/go/1.23.3/libexec/src/runtime/panic.go:785 +0x124",
          "translated/github.com/kmrgirish/gosim/internal_/tests/behavior_test.ImplTestLogForPrettyTest.func1()",
          "  /Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/tests/behavior/log_gosim_test.go:60 +0xe0",
          "translated/github.com/kmrgirish/gosim/internal_/simulation.(*Simulation).startMachine.func1()",
          "  /Users/jelle/hack/gosim/.gosim/translated/linux_arm64/github.com/kmrgirish/gosim/internal_/simulation/simulation_gosim.go:96 +0x4c",
          "github.com/kmrgirish/gosim/gosimruntime.(*goroutine).entrypoint(0x1400045de40, 0x14000461f50)",
          "  /Users/jelle/hack/gosim/gosimruntime/runtime.go:712 +0x60",
          "github.com/kmrgirish/gosim/gosimruntime.goroutineEntrypoint()",
          "  /Users/jelle/hack/gosim/gosimruntime/runtime.go:705 +0x30",
          "github.com/kmrgirish/gosim/internal/coro.(*Coro).Start.func1(0x0?)",
          "  /Users/jelle/hack/gosim/internal/coro/coro_linkname.go:38 +0x24",
          "created by github.com/kmrgirish/gosim/internal/coro.(*Coro).Start in goroutine 5",
          "  /Users/jelle/hack/gosim/internal/coro/coro_linkname.go:36 +0x70",
          ""
        ]
